### PR TITLE
ModelHub BQ: Use `dt.strftime` instead of `dt.sql_format`

### PR DESCRIPTION
**Pull request by KathiaBarahona**
`sql_format` is going to be deprecated, now ModelHub.time_agg uses `strftime` instead. Also changed `TIME_DEFAULT_FORMAT='YYYY-MM-DD HH24:MI:SS.MS` to `TIME_DEFAULT_FORMAT = '%Y-%m-%d %H:%M:%S.%f'`. Now, all tests in `test_model_hub_unique_sessions.py` are passing for BigQuery.

Will remove ignore of deprecation in another PR

### CODE REVIEW

File: modelhub/modelhub/modelhub.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def time_agg(self, data: bach.DataFrame, time_aggregation: str = None) -> bach.S
         """
```
Comment (thijs-obj) [inline modelhub/modelhub/modelhub.py:181]: The docstring on line 176 to line 179 still talks about the old format

**Review by thijs-obj — COMMENTED** [review]
Additional comment: We should undo these changes:
- https://github.com/objectiv/objectiv-analytics/pull/797/files#diff-9a656cd768faaf0dc0184535c74d2487ee1fe2113e104200479720f6d2705614R10
- https://github.com/objectiv/objectiv-analytics/pull/797/files#diff-2bad87ebc58328b17ae041cebf1e4d9bfab5b8a19a9455b9e798b0badfe9def9R73

edit: just noticed that the description said it will be fixed in another PR. That's fine too

File: modelhub/modelhub/modelhub.py (hunk 1 above)
Comment (KathiaBarahona) [reply modelhub/modelhub/modelhub.py:181]: good catch

Comment (KathiaBarahona) [reply modelhub/modelhub/modelhub.py:181]: Fixed here https://github.com/objectiv/objectiv-analytics/pull/842/commits/2fe0c61119023d26749fad0923a9eb617ee1e499

File: notebooks/basic-user-intent.ipynb [hunk 2]
```diff
@@ -71,11 +99,15 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "da4ec1a0",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
```
Comment (KathiaBarahona) [inline notebooks/basic-user-intent.ipynb:104]: :thinking:

**Review by jansentom — APPROVED** [review]
LGTM, thanks

**Pull request merged**